resources/functionMap.php

```diff
@@ -1606,8 +1606,8 @@
 'DomNode::add_namespace' => ['bool', 'uri'=>'string', 'prefix'=>'string'],
 'DomNode::append_child' => ['__benevolent<DOMNode|false>', 'newnode'=>'DOMNode'],
 'DOMNode::appendChild' => ['__benevolent<DOMNode|false>', 'newnode'=>'DOMNode'],
-'DOMNode::C14N' => ['string', 'exclusive='=>'bool', 'with_comments='=>'bool', 'xpath='=>'array', 'ns_prefixes='=>'array'],
-'DOMNode::C14NFile' => ['int', 'uri='=>'string', 'exclusive='=>'bool', 'with_comments='=>'bool', 'xpath='=>'array', 'ns_prefixes='=>'array'],
+'DOMNode::C14N' => ['__benevolent<string|false>', 'exclusive='=>'bool', 'with_comments='=>'bool', 'xpath='=>'array|null', 'ns_prefixes='=>'array|null'],
```

Comment:
Looking at https://www.php.net/manual/fr/domnode.c14n.php and https://www.php.net/manual/fr/domnode.c14nfile.php it also return false.

I used benevolent union to be consistent with other methods